crates/sui-json-rpc/src/api.rs

```diff
@@ -388,6 +395,8 @@ pub trait RpcTransactionBuilder {
         gas: Option<ObjectID>,
         /// the gas budget, the transaction will fail if the gas cost exceed the budget
         gas_budget: u64,
+        // Whether this is a regular transaction or a Dev Inspect Transaction
```

Comment:
Could you document what the default is, (when it's `None`)?  Or better yet, I might suggest not having this be optional -- i.e. if `None` behaves exactly like one of the `SuiTransactionBuilderMode`s, then make it explicit.

```suggestion
        /// Whether this is a regular transaction or a Dev Inspect Transaction
```

Reply:
Good call out for documenting the default value! I added it as an `optional` mainly for backward-compatibility, otherwise this will become a breaking change